app/services/nebraska/total_time_in_care_calculator.rb

```diff
@@ -19,6 +19,17 @@ def call
 
     def calculate_total_time
       service_day.update!(total_time_in_care: schedule_or_duration)
+      day_duration = calculate_full_time_and_part_time
+      service_day.update!(full_time: day_duration[:full_time], part_time: day_duration[:part_time])
+    end
+
+    def calculate_full_time_and_part_time
+      state = State.find_by(name: 'Nebraska')
+      time_engine = TimeConversionEngine.new(service_day: service_day, state: state)
+      time_rule = time_engine.call
+      return { full_time: 0, part_time: 1 } if time_rule.first.name == 'Partial Day Nebraska'
```

Comment:
the service should return the matching result. We should not do the .first here

Reply:
Fixed with the latest changes